cypress/e2e/register/formValidation.cy.js

```diff
@@ -0,0 +1,18 @@
+describe("Test registration form", () => {
+    beforeEach(() => {
+        cy.visit("/auth/register");
+    });
+    it('test user registration', () => {
+        cy.get('#input-email').as('email').click();
+        cy.get('body').click();
+        cy.get('@email').next().should('have.text', ' Email is required! ');
+
+        cy.get('#input-password').as('password').click();
+        cy.get('@email').click();
+        cy.get('@password').next().should('have.text', ' Password is required! ');
+        
+        cy.get('#input-re-password').as('re-password').click();
+        cy.get('@email').click();
+        cy.get('@re-password').next().should('have.text', ' Password confirmation is required! ');
```

Comment:
small improvement here
```suggestion
     function verifyFormElements(selector, errorMessage) {
            cy.get(selector).as('selector').click();
            cy.get('#title').click();
            cy.get('@selector').next().should('have.text', errorMessage);      
        }

        verifyFormElements('#input-email', ' Email is required! ');
        verifyFormElements('#input-password', ' Password is required! ');
        verifyFormElements('#input-re-password', ' Password confirmation is required! ');
```